install/deploy/base/role.yaml

```diff
@@ -0,0 +1,105 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: odh-dashboard # {"$openapi":"DASHBOARD_NAME"}
+rules:
+- apiGroups:
+  - route.openshift.io
+  resources:
+  - routes
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - kfdef.apps.kubeflow.org
+  resources:
+  - kfdefs
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ''
+  verbs:
+  - create
+  - delete
+  - get
+  - patch
+  - update
+  - watch
+  resources:
+  - configmaps
+  - secrets
+- apiGroups:
+  - batch
+  verbs:
+  - create
+  - delete
+  - get
+  - patch
+  - update
+  - watch
+  resources:
+  - cronjobs
+  - jobs
+- apiGroups:
+  - image.openshift.io
+  verbs:
+  - create
+  - get
+  - patch
+  resources:
+  - imagestreams
+- apiGroups:
+  - applications.console.openshift.io
+  resources:
+  - odhapplications
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - applications.console.openshift.io
+  resources:
+  - applications/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - applications.console.openshift.io
+  resources:
+  - odhapplications/status
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+  - documents.console.openshift.io
+  resources:
+  - odhdocuments
+  verbs:
+  - create
```

Comment:
Why does the dashboard need to create/edit the documents? Should it be enough to get/list/watch them?

Reply:
@vpavlin makes sense. I'll update the role.